plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/my-subscriptions.scss

```diff
@@ -157,3 +160,32 @@
 		margin-right: $grid-unit-10;
 	}
 }
+
+.woocommerce-marketplace__my-subscriptions__notices {
+	.components-notice {
+		margin-left: 0;
+		margin-right: 0;
+		background-color: #fff;
+		box-shadow: 0 2px 6px 0 rgba($gray-100, 0.05);
```

Comment:
If this value isn't coming from a design, I would be inclined to darken it a little bit – the notice is hard to notice on the page at the moment.

<img width="500" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1647564/1eb08431-804f-4e8f-945e-1d33da28b0fe">

Reply:
It was missing a border. It should look better now.
![Screenshot 2023-11-07 at 08 37 13](https://github.com/woocommerce/woocommerce/assets/1199991/e232a1a5-cf7e-4d42-bc8e-bcec87e063ab)